_i18n/pt_BR/factors/backwards-compatible-apis.md

```diff
@@ -0,0 +1,23 @@
+Apesar da maioria dos seus usuários atualizarem para a última versão dentro de poucas semanas, sempre existirá um grupo de usuários que não o fará. Isso pode ter inúmeras razões. Algumas vezes isso está relacionado com a versão do iOS que eles usam, versão essa que eles nem sempre podem atualizar dependendo do quão antigo o aparelho é.
+
+Você pode instalar e usar o Facebook Messenger num iPad da primeira geração (2010). Apesar de novas funcionalidades não serem suportadas, a funcionalidade principal ainda é disponibilizada graças ao versionamento de APIs.
+
+O conceito básico é que você não atualiza uma API existente, mas sim adiciona uma nova e deixa elas rodando em paralelo.
```

Comment:
Maybe `mas sim adiciona uma nova e mantém ambas rodando em paralelo.` instead of `mas sim adiciona uma nova e deixa elas rodando em paralelo.`